tests/baselines/reference/arrowFunctionContexts.js

```diff
@@ -116,7 +116,8 @@ var Base = (function () {
 var Derived = (function (_super) {
     __extends(Derived, _super);
     function Derived() {
-        return _super.call(this, function () { return _this; }) || this;
+        var _this = _super.call(this, function () { return _this; }) || this;

```

Comment:
ES2015 doesn't have a bunch of existing TypeScript code that evaluates this in a callback before the super() call returns.